.github/workflows/pull-request.yml

```diff
@@ -16,11 +16,21 @@ jobs:
   build_base:
     name: Build base env
     runs-on: ubuntu-latest
+    outputs:
+      target_env: ${{ steps.set_env.outputs.TARGET_ENV }}
     steps:
       - uses: actions/checkout@v4
         with:
           ref: ${{ github.base_ref }}
 
+      - name: Ensure SHORT_SHA is Set
```

Comment:
Why not just re-use the existing action `./.github/actions/short-sha` here?
Cause here you're checking if its been set, before we've called the action to set it. So it's never going to have been set at this point.